internal/kv/rocksdb/rocksdb_kv.go

```diff
@@ -33,6 +33,7 @@ func NewRocksdbKV(name string) (*RocksdbKV, error) {
 		return nil, errors.New("rocksdb name is nil")
 	}
 	bbto := gorocksdb.NewDefaultBlockBasedTableOptions()
+	bbto.SetPinL0FilterAndIndexBlocksInCache(true)
```

Comment:
bbto.SetPinL0FilterAndIndexBlocksInCache(true） should it be?

Reply:
CacheIndexAndFilterBlocks is set in [pr8851](https://github.com/milvus-io/milvus/pull/8851);
Set PinL0FilterAndIndexBlocksInCache because of  [this](https://github.com/facebook/rocksdb/wiki/Memory-usage-in-RocksDB#indexes-and-filter-blocks)